menu/src/state/vehiclemode.state.ts

```diff
@@ -0,0 +1,13 @@
+import { atom, useRecoilState } from "recoil";
+
+export enum VehicleMode {
+  SPAWN = "spawn",
+  DELETE = "delete",
+}
+
+const vehicleMode = atom({
+  key: "vehicleModeState",
+  default: VehicleMode.SPAWN,
+});
+
+export const useVehicleMode = () => useRecoilState(vehicleMode);
```

Comment:
We don't really need this to be a remembered state probably. Fine to just be reset on mount

Reply:
This is how other multi-choice options are mananged, healmode, teleportmode and playermode so I kept that the same for this.

Reply:
Yea we don't need that for those either, I implemented it with player mode as that is a persistent mode for players, the other ones are somewhat unnecessary.